helm/cluster-api/templates/crd-install/crd-ciliumnetworkpolicy.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.crds.install }}
+{{- if .Values.ciliumNetworkPolicy.enabled }}
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: {{ include "cluster-api.crdInstall" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  annotations:
+    # create hook dependencies in the right order
+    "helm.sh/hook-weight": "-2"
+    {{- include "cluster-api.CRDInstallAnnotations" . | nindent 4 }}
+  labels:
+    app.kubernetes.io/component: {{ include "cluster-api.crdInstall" . | quote }}
+    {{- include "cluster-api.selectorLabels" . | nindent 4 }}
+    role: {{ include "cluster-api.CRDInstallSelector" . | quote }}
+spec:
+  endpointSelector:
+    matchLabels:
+      app.kubernetes.io/component: {{ include "cluster-api.crdInstall" . | quote }}
+      {{- include "cluster-api.selectorLabels" . | nindent 6 }}
+  egress:
+    - toEntities:
+        - kube-apiserver
+{{- end }}
+{{- end }}
```

Comment:
Is this file also generated? I was expecting to see it being added to the `config` folder too

Reply:
I don't think this directory is generated. It's a mess.